packages/api/src/command/medical/patient/__tests__/store-query-init.test.ts

```diff
@@ -0,0 +1,99 @@
+import { PatientModel } from "../../../../models/medical/patient";
+import { mockStartTransaction } from "../../../../models/__tests__/transaction";
+import { makePatientModel } from "../../../../models/medical/__tests__/patient";
+
+import { storeQueryInit, StoreQueryParams } from "../query-init";
+import {
+  pdParams,
+  dqParams,
+  cqParams,
+  mockedPatientAllProgresses,
+  documentQueryProgress,
+  patientDiscovery,
+  consolidatedQuery,
+} from "./store-query-cmd";
+
+let patientModel: PatientModel;
+
+let patientModel_update: jest.SpyInstance;
+let patientModel_findOne: jest.SpyInstance;
+jest.mock("../../../../models/medical/patient");
+
+beforeEach(() => {
+  patientModel = makePatientModel();
+  mockStartTransaction();
+  patientModel_findOne = jest.spyOn(PatientModel, "findOne").mockResolvedValue(patientModel);
+  patientModel_update = jest.spyOn(patientModel, "update");
+});
+
+afterEach(() => {
+  jest.restoreAllMocks();
+});
+
+const checkPatientUpdateWith = (params: StoreQueryParams) => {
+  expect(patientModel_update).toHaveBeenCalledWith(
+    expect.objectContaining({
+      data: expect.objectContaining(params.cmd),
+    }),
+    expect.anything()
+  );
+};
+
+const checkPatientDataUpdatedWithMockedPatient = async (params: StoreQueryParams) => {
+  patientModel_findOne.mockResolvedValueOnce(mockedPatientAllProgresses);
+  patientModel_update = jest.spyOn(mockedPatientAllProgresses, "update");
+
+  await storeQueryInit(params);
```

Comment:
This seems to be the function that we're testing, right? It's hard to understand tests if it's not called within the test. I think it's fine to have setup and check functions, but I think the function being tested should be called directly within the test code.

Reply:
Ill just add the logic in the function back into the test